CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # TAMR AWS EMR module
 
+## v6.0.0 - July 13th, 2021
+* Removes creation of security groups except for internal use within the cluster
```

Comment:
Make sure to include all the changes, including the new nested module and updated examples.

Being explicit about the inputs/outputs that were removed/added in this release help the user of the module upgrade.

Reply:
Done